src/formatters/codeFrameFormatter.ts

```diff
@@ -97,4 +98,8 @@ export class Formatter extends AbstractFormatter {
 
         return `${outputLines.join("\n")}\n`;
     }
+
+    public sortFailures(failures: RuleFailure[]): RuleFailure[] {
+        return failures.slice().sort(RuleFailure.compare);
```

Comment:
Personally, I would not write a method for this simple statement. If you want to keep the method, it makes sense to move it to `AbstractFormatter` as you already noted. Works for me either way.